### PR TITLE
Update Global Environment Variables

### LINKED ISSUE #1577: Update Env Vars with OL + Airflow defaults
### Context

We historically haven't wanted to document system environment variables that we set by default but can be overridden, because some of them showcase our learnings from customers. With that said, because these can be set in different places (Runtime, Harmony, Airflow Operator) -- it's becoming hard to track them down internally and could just as well be helpful for customers.

https://docs.astronomer.io/astro/platform-variables

### Ask

- [ ] Add OL system Env Vars that _cannot_ be overridden (need to confirm with @conorbev)
        - `OPENLINEAGE_UR =`
       - `OPENLINEAGE_API_KEY = `
- [ ] Add a new section in this doc called **Defaults** with a 1-2 sentence description that state that these can be overridden by the user and are smart defaults
- [ ] Add a few env vars that we've missed thus far:
      - AIRFLOW__CELERY__STALLED_TASK_TIMEOUT=600
      - AIRFLOW_CORE_PARALLELISM =
      - [Research Others]

**Note:** Users can override the OL namespace to something like "prod" or "dev" and we need more prescriptive guidelines around that

cc: @cmarteepants @ashb I'd like to confirm with you two which things we set and where to make sure it's accurate

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1577

### CODE REVIEW

**Comment by kwcanuck** [comment]
@jwitz ready for your review. I'm not sure I captured all of Paola's edit requests.

File: astro/platform-variables.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ If you need to set one of these variables for a particular use case, contact [As
 | `AIRFLOW__WEBSERVER__NAVBAR_COLOR`         | The color of the main navigation bar in the Airflow UI                                                               | `#4a4466`                               |
 | `AIRFLOW__WEBSERVER__EXPOSE_CONFIG`        | Exposes the **Configuration** tab of the Airflow UI and hides sensitive values                                       |  `NON-SENSITIVE-ONLY`                   |
 | `AWS_SECRET_ACCESS_KEY`                    | The key secret for accessing Astro's managed S3 bucket¹                                                              | `<s3-aws-access-key-secret>`            |
-| `AWS_ACCESS_KEY_ID`                        | The key ID for accessing Astro's managed S3 bucket¹                                                                  | `<s3-aws-access-key-id>`                |
+| `OPENLINEAGE_URL`                    |  The URL for your Astro base domain.                                                            |       `https://<your-astro-base-domain>`      |
+ `OPENLINEAGE_API_KEY`                    |  Your OpenLineage API key                                                            |       `<your-lineage-api-key>`      |
```
Comment (jwitz) [inline astro/platform-variables.md:52]:
```suggestion
 | `OPENLINEAGE_API_KEY`                    |  Your OpenLineage API key                                                            |       `<your-lineage-api-key>`      |
```

File: astro/platform-variables.md [hunk 2]
```diff
@@ -48,7 +48,8 @@ If you need to set one of these variables for a particular use case, contact [As
 | `AIRFLOW__WEBSERVER__NAVBAR_COLOR`         | The color of the main navigation bar in the Airflow UI                                                               | `#4a4466`                               |
 | `AIRFLOW__WEBSERVER__EXPOSE_CONFIG`        | Exposes the **Configuration** tab of the Airflow UI and hides sensitive values                                       |  `NON-SENSITIVE-ONLY`                   |
 | `AWS_SECRET_ACCESS_KEY`                    | The key secret for accessing Astro's managed S3 bucket¹                                                              | `<s3-aws-access-key-secret>`            |
-| `AWS_ACCESS_KEY_ID`                        | The key ID for accessing Astro's managed S3 bucket¹                                                                  | `<s3-aws-access-key-id>`                |
+| `OPENLINEAGE_URL`                    |  The URL for your Astro base domain.                                                            |       `https://<your-astro-base-domain>`      |
```
Comment (jwitz) [inline astro/platform-variables.md:51]:
```suggestion
| `OPENLINEAGE_URL`                    |  The URL for your Astro lineage backend. The destination for lineage data sent from external systems to the OpenLineage API.                                                        |       `https://astro-<your-astro-base-domain>.datakin.com`      |
```

Comment (jwitz) [reply astro/platform-variables.md:51]: Based on [this discussion](https://astronomer.slack.com/archives/C03RTC9BQ4F/p1669998110787409?thread_ts=1669993091.471859&cid=C03RTC9BQ4F) how the URL is structured in some of our [code examples](https://github.com/astronomer/live-maps/blob/ee2f752159a44e80598b6d29b2848fe630cff76a/airflow/Dockerfile), I'm wondering if this is the correct format?

File: astro/runtime-image-architecture.md [hunk 3]
```diff
@@ -48,6 +48,16 @@ This table lists Astro Runtime releases and their associated Apache Airflow vers
 Each Runtime version in a given minor series supports only a single version of Apache Airflow. For specific version compatibility information, see [Runtime release notes](runtime-release-notes.md).
 :::
 
+## Default environment variables
+
+The following table list the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set by Astronomer, see [Global environment variables](platform-variables.md).
```
Comment (jwitz) [inline astro/runtime-image-architecture.md:53]:
```suggestion
The following table lists the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set on the Astro data plane, see [Global environment variables](platform-variables.md).
```

File: astro/runtime-image-architecture.md [hunk 4]
```diff
@@ -48,6 +48,16 @@ This table lists Astro Runtime releases and their associated Apache Airflow vers
...
+
+| Environment Variable                       | Description                                                                                                          | Value                                   |
+| ------------------------------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------- |
+| `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled. The default is zero.                                                                 | `integer`                |
+| `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment. The default is 32.                                                              | `string`            |
```
Comment (jwitz) [inline astro/runtime-image-architecture.md:58]:
```suggestion
| `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment.                                                               | `32`           |
```

File: astro/runtime-image-architecture.md [hunk 5]
```diff
@@ -48,6 +48,16 @@ This table lists Astro Runtime releases and their associated Apache Airflow vers
...
 :::
 
+## Default environment variables
+
+The following table list the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set by Astronomer, see [Global environment variables](platform-variables.md).
+
+| Environment Variable                       | Description                                                                                                          | Value                                   |
+| ------------------------------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------- |
+| `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled. The default is zero.                                                                 | `integer`                |
```
Comment (jwitz) [inline astro/runtime-image-architecture.md:57]:
```suggestion
| `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled.  | `0`   |
```

File: astro/runtime-image-architecture.md (hunk 4 above)
Comment (jwitz) [reply astro/runtime-image-architecture.md:58]: I don't think the value types are important here, so I replaced them with the default values which seems to be the core piece of information this table should show. Value types can be inferred from the default values themselves, at least in the case of these integer values.

**Review by jwitz — APPROVED** [review]

File: astro/runtime-image-architecture.md [hunk 6]
```diff
@@ -48,6 +48,16 @@ This table lists Astro Runtime releases and their associated Apache Airflow vers
 Each Runtime version in a given minor series supports only a single version of Apache Airflow. For specific version compatibility information, see [Runtime release notes](runtime-release-notes.md).
 :::
 
+## Default environment variables
+
+The following table lists the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set on the Astro data plane, see [Global environment variables](platform-variables.md).
```
Comment (paolaperaza) [inline astro/runtime-image-architecture.md:53]: Link to env vars doc as well/modify sentence to say, "To override..."

**Review by paolaperaza — APPROVED** [review]

File: astro/runtime-image-architecture.md [hunk 7]
```diff
@@ -48,6 +48,16 @@ This table lists Astro Runtime releases and their associated Apache Airflow vers
...
+
+| Environment Variable                       | Description                                                                                                          | Value                                   |
+| ------------------------------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------- |
+| `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled.  | `0`   |
+| `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment.                                                               | `32`           |
+
+
```
Comment (paolaperaza) [inline astro/runtime-image-architecture.md:60]: Extra space

Comment (kwcanuck) [reply astro/runtime-image-architecture.md]: Fixed

File: astro/runtime-image-architecture.md (hunk 6 above)
Comment (kwcanuck) [reply astro/runtime-image-architecture.md]: Added the text and link.

**Pull request merged**